client/app/modules/series/root/detailSerieFS/SeassonSF.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component, Fragment } from 'react';
+
+class SeassonSF extends Component {
+    
+    constructor(props) {
+        super(props);
+        this.state = {isOpen: false};
+    }
+
+    toogle() {
+        this.setState({ isOpen: !this.state.isOpen});
```

Comment:
Always use the next notation: this.setState({ ...state, isOpen: !this.state.isOpen });